src/vue-wrapper.ts

```diff
@@ -114,8 +114,12 @@ export class VueWrapper<T extends ComponentPublicInstance>
 
   findComponent(selector: FindComponentSelector): VueWrapper<T> | ErrorWrapper {
     if (typeof selector === 'object' && 'ref' in selector) {
-      return createWrapper(null, this.vm.$refs[selector.ref] as T)
+      const result = this.vm.$refs[selector.ref]
+      return result
+        ? createWrapper(null, this.vm.$refs[selector.ref] as T)
```

Comment:
would this be possible?

```suggestion
        ? createWrapper(null, result as T)
```